src/commcare_cloud/ansible/roles/bootstrap-users/tasks/main.yml

```diff
@@ -61,13 +61,6 @@
   authorized_key: user=ansible state=present key="{{ lookup('file', authorized_keys_dir ~ item ~ '.pub') }}"
   with_items: '{{ dev_users.present }}'
 
-- name: De-authorize ansible & cchq pubkey  # todo: remove after applied to all envs
```

Comment:
How come we can get rid of this?

Reply:
I did this in this commit: https://github.com/dimagi/commcare-cloud/pull/2013/commits/c538dd541678d3dd3bfbd10c1c08c4b096a96ac8. This task is a cleanup task we needed to run only because previously we had created ssh key logins for `cchq` and `ansible`, and we later decided we only want people to be able to log in with their own ssh keys, so we used this task to remove these shared public keys from the respective `.ssh/authorized_keys` files. Now that it's been applied to all envs, we can remove the taks, as in the todo comment above, "remove after applied to all envs".